marlowe-playground-server/app/FlowChartGenerator.hs

```diff
@@ -17,16 +17,16 @@ import qualified ZeroCouponBond as ZeroCouponBond
 main :: IO ()
 main = do
     putStrLn "===== SIMPLE ESCROW ====="
-    putStrLn $ toMermaid Escrow.contract
+    putStrLn $ toMermaid Escrow.escrow
```

Comment:
I had to change `contract` => for the name of the contract as the Marlowe extended package is now exporting a contract identifier